e2e/wdio/platform/tests/split-menu-button.e2e-spec.ts

```diff
@@ -1,213 +1,188 @@
 import { SplitMenuButtonPo } from '../pages/split-menu-button.po';
-import { browser } from 'protractor';
 import SMBData from '../fixtures/appData/split-menu-button-page-contents';
-import { getText, getValueOfAttribute, hoverMouse } from '../../helper/helper';
 import { webDriver } from '../../driver/wdio';
 
-describe('Split menu button test suite', function() {
+describe('Split menu button test suite', () => {
     const spMenuBtnPage = new SplitMenuButtonPo();
 
-    beforeAll(async () => {
-        await spMenuBtnPage.open();
+    beforeAll(() => {
+        spMenuBtnPage.open();
     });
 
     afterEach(() => {
         webDriver.refreshPage();
     });
 
     it('should check drop-down arrow menu functionality', async () => {
-        const dropdownArrowBtnArr = await spMenuBtnPage.arrowBtnArr;
+        const dropdownArrowBtnArr = browser.$$(spMenuBtnPage.arrowBtnArr);
+
+        for (let i = 0; i < dropdownArrowBtnArr.length; i++) {
+            dropdownArrowBtnArr[i].click();
+            expect(webDriver.waitForDisplayed(spMenuBtnPage.menuOverlay));
+            dropdownArrowBtnArr[i].click();
+        }
 
-        dropdownArrowBtnArr.forEach(async element => {
-            await element.click().then(async () => {
-                await expect(await spMenuBtnPage.menuOverlay.isDisplayed()).toBe(true);
-            });
-            await element.click();
-        });
     });
 
     it('should check each split btn has main and arrow btns', async () => {
-        const dropdownArrowBtnCount = await spMenuBtnPage.arrowBtnArr.count();
-        const mainBtnCount = await spMenuBtnPage.mainBtnArr.count();
+        const dropdownArrowBtnCount = $$(spMenuBtnPage.arrowBtnArr).length;
+        const mainBtnCount = $$(spMenuBtnPage.mainBtnArr).length;
```

Comment:
maybe better to use the method from the helper "getElementArrayLength" for such situations?